Corona-Warn-App/src/main/res/layout/fragment_greencertificate_details.xml

```diff
@@ -0,0 +1,334 @@
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="24dp"
+                    android:text="@string/green_certificate_attribute_state_of_testing" />
+
+                <TextView
+                    android:id="@+id/test_center"
+                    style="@style/body1"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="4dp"
+                    android:text="Germany"
+                    tools:text="Germany" />
+
+                <TextView
+                    style="@style/body1Medium"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="24dp"
+                    android:text="@string/green_certificate_attribute_certificate_issuer" />
+
+                <TextView
+                    android:id="@+id/test_country"
+                    style="@style/body1"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="4dp"
+                    android:text="G05930482748454836478695764787840"
+                    tools:text="G05930482748454836478695764787840" />
+
+
+                <TextView
+                    style="@style/body1Medium"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="24dp"
+                    android:text="@string/green_certificate_attribute_certificate_id" />
+
+                <TextView
+                    android:id="@+id/certificate_issuer"
+                    style="@style/body1"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="4dp"
+                    android:text="05930482748454836478695764787840"
+                    tools:text="05930482748454836478695764787840" />
+
+
+            </LinearLayout>
+
+            <LinearLayout
+                style="@style/Card.Greencertificate"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="24dp"
+                android:layout_marginTop="8dp"
+                android:orientation="vertical"
+                android:padding="16dp">
+
+                <TextView
+                    android:id="@+id/travel_notice_german"
+                    style="@style/body2"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="4dp"
+                    android:text="@string/green_certificate_attribute_certificate_travel_notice_german"/>
```

Comment:
Link should be clickable in this text ?
<img width="320" alt="Screenshot 2021-05-31 at 14 37 49" src="https://user-images.githubusercontent.com/25054729/120194353-cb903d80-c21d-11eb-8b2f-1c0a4f60d96d.png">

Reply:
is done now :)